src/unraphael/dash/pages/4_compare.py

```diff
@@ -117,7 +121,7 @@ def align_images_widget(*, base_image: ImageType, images: list[ImageType]) -> li
             )
         )
 
-    return res
+    return res, align_method, motion_model
```

Comment:
Can we refactor this in a way that avoids returning a tuple? This makes the code difficult to maintain.